python/paddle/fluid/tests/unittests/test_conv2d_op.py

```diff
@@ -246,8 +243,10 @@ def init_op_type(self):
         self.op_type = "conv2d"
 
 
-class TestFP16CUDNN(TestCUDNN):
-    def init_data_type(self):
+class TestFP16CUDNN(TestConv2dOp):
+    def init_op_type(self):
+        self.use_cudnn = True
+        self.op_type = "conv2d"
```

Comment:
`self.op_type` should be placed in `setUp`, because the unit tests of `test_conv2d_op.py` is all about `conv2d` and the difference is just `op_kernels`.

Reply:
Good point! Done.